core/src/test/java/io/jenkins/pluginhealth/scoring/repository/ScoreRepositoryTest.java

```diff
@@ -0,0 +1,87 @@
+package io.jenkins.pluginhealth.scoring.repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import hudson.util.VersionNumber;
+import io.jenkins.pluginhealth.scoring.AbstractDBContainerTest;
+import io.jenkins.pluginhealth.scoring.model.Plugin;
+import io.jenkins.pluginhealth.scoring.model.Score;
+
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+
+import java.time.ZonedDateTime;
+
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+@DataJpaTest
+public class ScoreRepositoryTest extends AbstractDBContainerTest {
+
+    @Autowired
+    private ScoreRepository repository;
+
+    @Autowired private TestEntityManager entityManager;
+
+    @Test
+    void shouldBeAbleToDeleteOldScores() {
+        final Plugin plugin1 = new Plugin("plugin-1", new VersionNumber("1.0"), "scm", ZonedDateTime.now());
+        entityManager.persist(plugin1);
```

Comment:
You need to assign the value returned by the `entityManager.persist` to the variable `plugin1`

```suggestion
        final Plugin plugin1 = entityManager.persist(new Plugin("plugin-1", new VersionNumber("1.0"), "scm", ZonedDateTime.now()));
```

And the same for all other persisted values.